deployment/configMigration.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+from clusterObjectModel import forward_compatibility
+import shutil
+import yaml
+import os
+import sys
+print("This script is used for migrating config from v0.8 to v0.10!")
```

Comment:
logger

Reply:
Hi ydye,
It's a one off script, so I try to make it simple.

Reply:
ok